modules/installation-guide/partials/proc_logging-in-to-identity-provider.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc_logging-in-to-identity-provider_{context}"]
+== Logging in to {identity-provider}
```

Comment:
Why a second-level heading?

Reply:
Robert told me to add this procedure into modules where it is needed, so the second level matches the purpose.

Reply:
@MichalMaler, I don't understand this. A module/procedure should never have anything but a first-level heading in the beginning. If you need to include it in some weird doc structure, use `leveloffset` on the include statement.

Reply:
@rkratky  I Will patch this in my next MR tomorrow